conda_forge_tick/utils.py

```diff
@@ -380,6 +380,300 @@ def _render_meta_yaml(text: str, for_pinning: bool = False, **kwargs) -> str:
         raise
 
 
+def parse_recipe_yaml(
+    text: str,
+    for_pinning=False,
+    platform=None,
+    arch=None,
+    cbc_path=None,
+    log_debug=False,
+    use_container: bool | None = True,
```

Comment:
This default should be None too. I think this is a bug I put into these functions more generally. Let's change this one and I will submit a PR fixing any others I can find.

Reply:
Done